### PR TITLE
Task #178236 fix: event detail repo updation

### PR DESCRIPTION
**Pull request by AnkitaMaheshChavan**

### CODE REVIEW

File: projects/event-library/src/lib/events/components/advance-event-detail/advance-event-detail.component-copy.html [hunk 1]
```diff
@@ -0,0 +1,245 @@
+<!-- <div class="">
+    <div>
```
Comment (paritshivani) [inline projects/event-library/src/lib/events/components/advance-event-detail/advance-event-detail.component-copy.html:2]: Do we need this file??
@AnkitaMaheshChavan

File: projects/event-library/src/lib/events/components/advance-event-detail/advance-event-detail.component-copy.html [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+                            <!-- new code -->
+                            <!-- <div _ngcontent-klo-c24="" class="avatar-content ng-star-inserted" style="text-align: center; border-radius: 50%; border: 1px solid rgb(232, 232, 232); text-transform: uppercase; color: rgb(51, 51, 51); background-color: rgb(255, 255, 255); font-style: normal; font-variant: normal; font-weight: normal; font-stretch: normal; font-size: 24px; line-height: 48px; font-family: inherit; height: 48px; width: 48px;">{{this.eventCreatorInfo.firstName[0]}}</div> -->
+                        </div>
+                        <div class="sb-card-image info mr-10 ml-20 mt-20 mu-20 mb-20" >
+                            <b><p class="mb-2">{{this.userName}}</p></b>
+                            <p class="mb-2 fs-6">Designation</p>
+                            <p class="mb-2 fs-6">{{this.eventConfig.role}}</p>
+
+                            <!-- new code -->
+                            <!-- <b><p class="mb-2">{{this.eventCreatorInfo.firstName}}</p></b>
```
Comment (paritshivani) [inline projects/event-library/src/lib/events/components/advance-event-detail/advance-event-detail.component-copy.html:147]: remove unnecessary commented codes

File: projects/event-library/src/lib/events/components/advance-event-detail/advance-event-detail.component-copy.html (hunk 1 above)
Comment (AnkitaMaheshChavan) [reply projects/event-library/src/lib/events/components/advance-event-detail/advance-event-detail.component-copy.html:2]: resolved

File: projects/event-library/src/lib/events/components/advance-event-detail/advance-event-detail.component-copy.html (hunk 2 above)
Comment (AnkitaMaheshChavan) [reply projects/event-library/src/lib/events/components/advance-event-detail/advance-event-detail.component-copy.html:147]: resolved

File: projects/event-library/src/lib/events/components/cover-event-detail/cover-event-detail.component.ts [hunk 3]
```diff
@@ -1,67 +1,165 @@
...
+
+    var timezoneshort = this.timezoneCal.timeZoneAbbreviated();
+
+    this.eStart = this.datepipe.transform(this.eventDetailItem.startDate, 'longDate') + ', ' + this.datepipe.transform(startEventTime, 'HH:mm') + ' (' + timezoneshort + ')';
+    this.eEnd = this.datepipe.transform(this.eventDetailItem.endDate, 'longDate') + ', ' + this.datepipe.transform(endEventTime, 'HH:mm') + ' (' + timezoneshort + ')';
   }
 
-  upate(identifier, versionKey) {
-     this.router.navigate(['/event-post'], {
+  update_old(identifier, versionKey) {
```
[the remark this answers is not in the document]
Comment (AnkitaMaheshChavan) [reply projects/event-library/src/lib/events/components/cover-event-detail/cover-event-detail.component.ts:92]: resolved

File: projects/event-library/src/lib/events/components/join-event-button/join-event-button.component.ts [hunk 4]
```diff
@@ -10,37 +14,85 @@ import { EventService } from '../../services/event/event.service'
...
+      {
+        this.isStartDate1 = false;
+        this.isStartDate2 = false;
+      }
       this.isEnrollEvent();
       this.joinEvent();
     }
-}
+    
+    this.someDate = new Date( Date.now() + (1 * 60) * 1000 );
```
[the remark this answers is not in the document]
Comment (AnkitaMaheshChavan) [reply projects/event-library/src/lib/events/components/join-event-button/join-event-button.component.ts:88]: resolved

File: projects/event-library/src/lib/events/components/join-event-button/join-event-button.component.ts [hunk 5]
```diff
@@ -111,4 +261,19 @@ export class JoinEventComponent implements OnInit {
   openProviderLink(joinLink) {
     window.open(joinLink, "_blank");
   }
+
+  myTriggerFunction() {
```
[the remark this answers is not in the document]
Comment (AnkitaMaheshChavan) [reply projects/event-library/src/lib/events/components/join-event-button/join-event-button.component.ts:265]: Renamed to CounterZeroTriggerFunction
resolved

File: src/polyfills.ts [hunk 6]
```diff
@@ -1,3 +1,7 @@
+/***************************************************************************************************
+ * Load `$localize` onto the global scope - used if i18n tags appear in Angular templates.
+ */
+import '@angular/localize/init';
```
[the remark this answers is not in the document]
Comment (AnkitaMaheshChavan) [reply src/polyfills.ts:4]: required for to resolve issue -> "[Angular 9 introduced a global '$localize()' function that needs to be loaded](https://stackoverflow.com/questions/57953317/angular-9-introduced-a-global-localize-function-that-needs-to-be-loaded)"

File: src/app/home/home.component.html [hunk 7]
```diff
@@ -1,6 +1,6 @@
 <!-- <sb-event-detail [eventDetailItem]="eventItem" [userData]="userId"></sb-event-detail> -->
 
 
-<!-- <sb-cover-event-detail [eventDetailItem]="eventItem" [userData]="userId"></sb-cover-event-detail>
-<sb-join-event-button [eventDetailItem]="eventItem" [userData]="userId" [canUnenroll]="true"></sb-join-event-button>
-<sb-advance-event-detail [eventDetailItem]="eventItem"></sb-advance-event-detail> -->
\ No newline at end of file
+<sb-cover-event-detail [eventDetailItem]="eventItem"></sb-cover-event-detail>
```
[the remark this answers is not in the document]
Comment (AnkitaMaheshChavan) [reply src/app/home/home.component.html:4]: removed

File: projects/event-library/src/lib/events/components/join-event-button/join-event-button.component.ts [hunk 8]
```diff
@@ -111,4 +261,19 @@ export class JoinEventComponent implements OnInit {
...
+    this.isDisabled= false;
+    this.toShowCounter= false;
+  }
+
+  navtoedit()
```
[the remark this answers is not in the document]
Comment (AnkitaMaheshChavan) [reply projects/event-library/src/lib/events/components/join-event-button/join-event-button.component.ts:270]: resolved

File: projects/event-library/src/lib/events/components/event-detail/event-detail.component.html [hunk 9]
```diff
@@ -1,6 +1,5 @@
 <div class="event-wrapper">
-    <sb-cover-event-detail [eventDetailItem]="eventDetailItem" [userData]="userData"></sb-cover-event-detail>
-    <sb-join-event-button [eventDetailItem]="eventDetailItem" [userData]="userData" [canUnenroll]="canUnenroll"></sb-join-event-button>
+    <sb-cover-event-detail [eventDetailItem]="eventDetailItem"></sb-cover-event-detail>
```
[the remark this answers is not in the document]
Comment (AnkitaMaheshChavan) [reply projects/event-library/src/lib/events/components/event-detail/event-detail.component.html:2]: did similar as in repo https://github.com/paritshivani/nsdl-event-library/

**Pull request merged**